libkpod/config.go

```diff
@@ -0,0 +1,38 @@
+package libkpod
+
+// Config stores the configuration options for libkpod
+type Config struct {
+	Root                     string
+	RunRoot                  string
+	Storage                  string
+	StorageOptions           []string
+	LogDir                   string
+	Runtime                  string
+	RuntimeUntrustedWorkload string
+	DefaultWorkloadTrust     string
+	Conmon                   string
+	ConmonEnv                []string
+	CgroupManager            string
+	DefaultTransport         string
+	InsecureRegistries       []string
+	SignaturePolicyPath      string
+}
+
+// DefaultConfig returns a Config struct with all options set to the default
+func DefaultConfig() *Config {
```

Comment:
I don't know if we want to bother with this at first. I'd be fine with leaving libkpod consumers to configure everything themselves and manage defaults independently. @mrunalp WDYT?